docs/protocol/extension_sequence.md

```diff
@@ -28,31 +28,36 @@
 
 # Sequence Extension
 
-This document describes Triton's sequence extension.  The sequence
+This document describes Triton's sequence extension. The sequence
 extension allows Triton to support stateful models that expect a
-sequence of related inference requests.  Because this extension is
-supported, Triton reports “sequence” in the extensions field of its
-Server Metadata.
+sequence of related inference requests.
 
 An inference request can specify that it is part of a sequence using
 the “sequence_id” parameter in the request and by using the
 “sequence_start” and “sequence_end” parameters to indicate the start
 and end of sequences.
 
-- "sequence_id" : uint64 value that indicates which sequence a request
-  belongs to. All inference requests that belong to the same sequence
-  must use the same sequence ID. A sequence ID of 0 indicates the
+Because this extension is supported, Triton reports either "sequence"
```

Comment:
Sorry to be pedantic... but "sequence" must always be reported if any of sequence extension is supported. *Additionally* "sequence(string_id)" can be reported if server also supports sequence IDs as strings.

Reply:
Oh i see. I should have understood that, apologies.